base_local_planner: dwa_mobs_planner/DWAMobsPlannerROS
DWAMobsPlannerROS:
  # Robot Configuration Parameters
  acc_lim_x: 6.0
  acc_lim_y: 0.0
  acc_lim_theta: 4.0
  max_trans_vel: 0.45
  min_trans_vel: 0.05
  max_vel_x: 0.45
  min_vel_x: 0.05
  max_vel_y: 0.0
  min_vel_y: 0.0
  max_rot_vel: 2.0
  min_rot_vel: 0.0

  # Goal Tolerance Parameters
  yaw_goal_tolerance: 0.15
  xy_goal_tolerance: 0.15
  latch_xy_goal_tolerance: false

  # Forward Simulation Parameters
  sim_time: 4.0
  sim_granularity: 0.025
  vx_samples: 5
  vy_samples: 0
  vtheta_samples: 20
  controller_frequency: 2
  penalise_negetive_x: true

  # Trajectory Scoring Parameters
  path_distance_bias: 3.2
  goal_distance_bias: 24
  occdist_scale: 2.75
  forward_point_dist: 0.325
  stop_time_buffer: 0.2
  scaling_speed: 0.25
  max_scaling_factor: 0.2

  # Oscillation Prevention Parameters
  oscillation_reset_dist: 0.01

  # Global Plan Parameters
  prune_plan: true

  transform_tolerance: 2.0
  world_model: costmap
  simple_attractor: false
  meter_scoring: true
  holonomic_robot: false
  use_dwa: true
 


  
